README.md

```diff
@@ -1,2 +1,36 @@
-# Bamboo-plugin
-Leapwork Integration with Bamboo
+# Leapwork Integration
+This is Leapwork plugin for Bamboo
+
+# More Details
+Leapwork is a mighty automation testing system and now it can be used for running [smoke, functional, acceptance] tests, generating reports and a lot more in Bamboo. You can easily configure integration directly in Bamboo enjoying UI friendly configuration page with easy connection and test suites selection.
+
+# Features:
+ - Setup and test Leapwork connection in few clicks
+ - Run automated tests in your Bamboo build tasks
+ - Automatically receive test results
+ - Build status based tests results
+ - Generate a xml report file in JUnit format
+ - Write tests trace to build output log
+ - Smart UI
+ 
+# Installing
+- Use atlassian-sdk maven 8.0.7
+- Command: atlas-package 
+- Or download jar file of OBR file from Releases section.
+
+# Instruction
+1. Add Build "Leapwork Integration" to your job.
+2. Enter your Leapwork controller hostname or IP-address something like "win10-agent20" or "localhost".
```

Comment:
Leapwork controller API hostname

Reply:
hostname is same for both controller and api, its the port number that is different